### PR TITLE
Allow extending PATH from environment variables

### PR DESCRIPTION
**Pull request by acbramley**
You can't extend existing environment variables in docker-compose, and using a custom dockerfile just for that seems overkill. With this on GEL we can add:

```
- EXTRA_PATH=/data/api/bin
```

For better DX.

### CODE REVIEW

**Comment by kimpepper** [comment]
Nice! :ok_hand: Can we call it `$EXTRA_PATH` for brevity?

File: php/cli/Dockerfile [hunk 1]
```diff
@@ -4,6 +4,8 @@ ARG PHP_VERSION=7.2
 
 ENV PHP_MEMORY_LIMIT=256M
 
+ENV PATH=$PATH:$ADDITIONAL_PATH
```
Comment (acbramley) [inline php/cli/Dockerfile:7]:
```suggestion
ENV PATH=$PATH:$EXTRA_PATH
```

**Comment by nickschuch** [comment]
This is rad. I wonder if this pattern could be container wide.

Curious if we move it to the lot level base container and set that env for these containers:

```
$ grep -R PATH *
$ grep -R 'ENV PATH' *
kubebuilder/v1/Dockerfile:ENV PATH=$PATH:/usr/local/kubebuilder/bin
kubebuilder/v2/Dockerfile:ENV PATH /usr/local/kubebuilder/bin:$PATH
node/Dockerfile:ENV PATH /data/node_modules/.bin:$PATH
php/base/Dockerfile:ENV PATH "$PATH:/data/bin"
```

**Comment by acbramley** [comment]
Tested doing `ENV PATH=$PATH:$EXTRA_PATH` in `skpr/base` then `ENV EXTRA_PATH=/data/bin` in `php/base` but it doesn't work :(

**Pull request merged**